{{cookiecutter.project_slug}}/src/data/dataGenerator.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
```

Comment:
import find from 'loadash/find' is way faster 👍

Reply:
no need

Reply:
There is need, as mentioned, `lodash/find` is faster while compiling build.

Reply:
let's have a new PR to remove lodash